Packages/ohif-measurement-table/client/configuration/dataExchange.js

```diff
@@ -18,10 +19,11 @@ export const retrieveMeasurements = (patientId, timepointIds) => {
 export const storeMeasurements = (measurementData, timepointIds) => {
     OHIF.log.info('storeMeasurements');
 
-    // Here is where we should do any required data transformation and API calls
+    const studyInstanceUid = measurementData[Object.keys(measurementData)[0]][0].studyInstanceUid
 
-    // TODO: Write SR, STOW back to PACS
-    return Promise.resolve();
+    return stowSRFromMeasurements(measurementData).then( () => {
```

Comment:
Here we should check what type of server we are connected to and act accordingly.

A similar example is used for retrieving metadata here:

https://github.com/OHIF/Viewers/blob/ebe0a40f3ef5fa7d13c66e7076ddeb1f42e755b5/Packages/ohif-studies/imports/client/lib/retrieveStudyMetadata.js#L49

There are three potential scenarios:
- The server is DICOMWeb and we can make requests from the client (e.g. we have a Bearer token or are connecting to an insecure server)
- The server is DIMSE, so we have to make requests from the server
- The server is DICOMWeb, but we have to make requests from the server (e.g. if we are using HTTP basic auth and don't want to expose the user/pass to the client)

In both of the DICOMWeb cases, we can use STOW-RS. In the DIMSE case, we will need to write something to support C-STORE.

For now, we can hide the 'Save to PACS' button when we are using a DIMSE server, and we can get that functionality in another PR.